Cloud Application Manager/Getting Started/getting-started-login-quick-tour.md

```diff
@@ -94,12 +96,32 @@ ___
 
 ![getting-started-login-5.png](../../images/cloud-application-manager/getting-started-login-5.png)
 
-* Connect to a provider to orchestrate deployments. Click **New Provider** to add AWS, Azure, vSphere, Google Cloud, OpenStack, or CloudStack.
+* Connect to a provider to orchestrate deployments. Click **New** to add AWS, Azure, vSphere, Google Cloud, OpenStack, or CloudStack.
 * Locate a provider through search or by type.
 * Sync or delete provider accounts using Bulk Actions.
 
 ___
 
+**Catalog**
+
+![getting-started-login-9.png](../../images/cloud-application-manager/getting-started-login-9.png)
```

Comment:
Hey @dsimon1992, please use a more complete screenshot of the Catalog. I guess you took it from a local env, but please use production to take this one

Reply:
Updated the screenshot.